src/endpoints/tokens/token.service.ts

```diff
@@ -65,8 +65,15 @@ export class TokenService {
     return tokens.map(item => mergeObjects(new TokenDetailed(), item));
   }
 
-  async getTokenCount(): Promise<number> {
+  async getTokenCount(search: string | undefined): Promise<number> {
     let allTokens = await this.getAllTokens();
+
+    if (search !== undefined) {
```

Comment:
I think `if (search)` is enough for this condition.

Reply:
Thank you for the code-review.

Yes, that's true, if (search) is enough for this condition.

I've updated it and I've pushed a new commit.

Reply:
All good! Thank you for your contribution.